runtime/cfdumper/romdump.c

```diff
@@ -561,7 +561,7 @@ reportSuspectedPadding(J9PortLibrary *portLib, J9ROMClass *romClass, J9ROMClassG
 	} else if (nbytes > 6) {
 		nbytes = 6;
 	}
-	j9tty_printf(PORTLIB, "0x%p-0x%p [ %*.s", base + lastOffset, base + offset, 12 - 2*nbytes , "");
+	j9tty_printf(PORTLIB, "0x%p-0x%p [ %.*s", base + lastOffset, base + offset, 12 - (2 * nbytes), "");
```

Comment:
The string being used is `""` which is null terminated rather than a pascal/utf8 string.  It doesn't need the `.*s` modifier.

I'd need to see output from this running before merging the changes

Reply:
I believe this is specifying a width rather the length of the string.

Reply:
ah right. Technically I think it should be `%*s`, but I don't think it makes any difference when printing `""`, both result in the same output (which isn't true for other strings than `""`).
```
printf("abc%*scba\n", 3, "");
printf("abc%*.scba\n", 3, "");
```
```
abc   cba
abc   cba
```

Reply:
I preference it to simply remove the `.`. The meaning of `.*` (as used most places) is that the length argument specifies the number of characters to be read from the value argument; in this case there are no characters in the value argument, so `.*` should not be used.

Reply:
Updated.